src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,343 @@
+                System.out.println("  Nice! I've marked this task as done:");
+                System.out.println("   " + taskNum + "." + storeTasks.get(taskNum - 1));
+                System.out.println(" ____________________________________________________________");
+            }
+            // Update text file
+            String textLine = tasksText.get(taskNum - 1);
+            String[] text = textLine.split(" \\| ");
+            if (text[0].equals("T")) {
+                String newText = "T" + " | " + "1" + " | " + storeTasks.get(taskNum - 1).description;
+                tasksText.set(taskNum - 1, newText);
+            }
+            if (text[0].equals("D")) {
+                String newText = "D" + " | " + "1" + " | " + text[2] + " | " + text[3];
+                tasksText.set(taskNum - 1, newText);
+            }
+            if (text[0].equals("E")) {
+                String newText = "E" + " | " + "1" + " | " + text[2] + " | " + text[2];
+                tasksText.set(taskNum - 1, newText);
+            }
+            try {
+                updateFile(tasksText, filePath);
+            } catch (FileNotFoundException e) {
+                System.out.println("Folder not found");
+            }
+        } catch (IndexOutOfBoundsException e) {
+            throw new DukeException("You do not have any task(s)");
+        }
+    }
+
+    private static void deadlineCommand(String line, ArrayList<Task> storeTasks, String filePath,
+                                        ArrayList<String> tasksText) throws DukeException{
+        String[] words = line.split("/by");
+        words[0] = words[0].replace("deadline ", "");
+        storeTasks.add(new Deadline(words[0], words[1]));
+        System.out.println(" ____________________________________________________________");
+        System.out.println("  Got it. I've added this task:");
+        System.out.println("    " + storeTasks.get(storeTasks.size() - 1));
+        System.out.println("  Now you have " + storeTasks.size() + " tasks in the list.");
+        System.out.println(" ____________________________________________________________");
+
+        // Structuring text to write to file
+        String text = "D" + " | " + "0" + " | " + words[0] + "|" + words[1];
+        tasksText.add(text);
+        try {
+            updateFile(tasksText, filePath);
+        } catch (FileNotFoundException e) {
+            System.out.println("Folder not found");
+        }
+    }
+
+    private static void todoCommand(String line, ArrayList<Task> storeTasks, String filePath,
+                                    ArrayList<String> tasksText) throws DukeException {
+        try {
+            if (line.equals("todo")) {
+                throw new DukeException("The description of a todo cannot be empty.");
+            }
+
+            String[] words = line.split("todo ");
+            storeTasks.add(new ToDo(words[1]));
+            //storeTasks[numOfTasks] = new ToDo(words[1]);
```

Comment:
This seems like past code that is commented out when not needed....you may want to delete this. Do make full use of version control to keep track of past code.